### PR TITLE
Allow string values for tickCount

### PR DESCRIPTION
**Pull request by jmattheis**
See #19

### CODE REVIEW

File: tests/spec/valid/scales-bin.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ const spec: Spec = {
 
   "axes": [
     {"orient": "bottom", "scale": "xscale", "tickCount": 20, "zindex": 1},
-    {"orient": "left", "scale": "yscale", "tickCount": 5, "zindex": 1}
+    {"orient": "left", "scale": "yscale", "tickCount": "hour", "zindex": 1}
```
Comment (domoritz) [inline tests/spec/valid/scales-bin.ts]: Did the original vega example change? If not, let's leave the examples as they are.

**Comment by domoritz** [comment]
Run `yarn format` to fix formatting issues.

File: tests/spec/valid/scales-bin.ts (hunk 1 above)
Comment (jmattheis) [reply tests/spec/valid/scales-bin.ts]: So where should I add a test for this?

Comment (domoritz) [reply tests/spec/valid/scales-bin.ts]: It's okay without a test in this case. We'll add one as we add more example specs.

**Comment by jmattheis** [comment]
@domoritz FYI removed the commit.

File: types/spec/axis.d.ts [hunk 2]
```diff
@@ -68,7 +69,7 @@ export interface Axis {
    * @minimum 0
    */
   tickSize?: number;
-  tickCount?: number | SignalRef;
+  tickCount?: number | SignalRef | TimeInterval;
```
Comment (pelotom) [inline types/spec/axis.d.ts]: For consistency, let’s put `SignalRef` last

**Pull request merged**